### DISCUSSION TITLE
Layout change transparency followed by black background

### DISCUSSION THREAD

**Discussion by madtraxx18** [marcokreeft87/formulaone-card#298] [Ideas]
Would you be able to make an option to have the backgrounds of the texts fully transparent? In the current situation the first line is transparent followed by a line with a black background, and so on.

This looks nice in the normal Dark theme of HA but when you use a different theme, it doesn't always match the Theme.

**Comment by marcokreeft87** [comment]
You can do that yourself with card mod. This is too taste specific to implement for now. Maybe if there are more requests for it

**Reply by madtraxx18** [reply]
well that might be way over my head, i'm glad I managed to get everything lined out in a nice manner :-)

here is a simple example of what I mean:

![image](https://github.com/marcokreeft87/formulaone-card/assets/16655593/ad4a2f1a-7bdd-4561-ba6c-e7c48c7549fc)

**Reply by marcokreeft87** [reply]
It's not that hard really. I'm sure the ha community can help you learn css
